### PR TITLE
pattern import log statement improved/signoff

### LINKED ISSUE #11422: [mesheryctl] `pattern import` command should have a proper log on successful import of pattern
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

### Current Behavior

When you use `mesheryctl pattern import`, the response received is often unclear and fails to convey the proper meaning.
### Desired Behavior

Enhance the log to include more meaningful information and clearly convey the correct message.
### Screenshots/Logs

```
mesheryctl pattern import -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/release-1.11/config/master/aws-k8s-cni.yaml -s kubernetes
remote hosted pattern request success
pattern file imported successfully.
ID of the pattern: e0bf8082
```
### Environment

- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by rahulrana701**
**Notes for Reviewers**

This PR fixes #11422

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
Thank you @rahulrana701 
Let's discuss this on the Meshery dev meeting. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk).

File: mesheryctl/internal/cli/root/pattern/import.go [hunk 1]
```diff
@@ -167,7 +167,7 @@ func importPattern(sourceType string, file string, patternURL string, save bool)
 		if err != nil {
 			return nil, utils.ErrRequestResponse(err)
 		}
-		utils.Log.Debug("remote hosted pattern request success")
+		utils.Log.Debug("Successfully fetched the pattern from the remote host")
```
Comment (Jougan-0) [inline mesheryctl/internal/cli/root/pattern/import.go:170]: Remove the word Successfully

**Review by leecalcote — CHANGES_REQUESTED** [review]
@rahulrana701, you left a "successfully" behind. :/

Also:

1. Please use the word "design" instead of pattern.
2. Include the name of the design or the filename of the design in the response to the user.

**Comment by rahulrana701** [comment]
@leecalcote sure

**Review by leecalcote — APPROVED** [review]
Excellent!

**Pull request merged**